update/update-version.md

```diff
@@ -1,139 +1,187 @@
 
-1. The user would have to update their code to consume version 4.x.x, and then update their schematics workspace to the version of their code that contains the updated module and click on `Generate plan` and make sure none of the VSIs will be recreated. The expected behaviour should be that the plans contains only naming updates, and no destroy / re-creates of any resources.
+    - Replace `<vpc-id1>` and `<vpc-region>` with the information that you copied earlier.
+    - If the VSIs are deployed in an account that is different from the account where the Schematics workspace is located, include the `-k` option and replace `<vpc-ibm-api-key>` with the IBM Cloud API key with access to the VPC.
 
-1. Click on `Apply plan`.
+    The script creates a job in the Schematics workspace.
 
-1. [Optional] If a schematics workspace job that was initiated through the script encounters an issue, you can undo any modifications made by running the script again with the -z flag. For example,
-    ```sh
-    bash schematics_update_v3.x.x_to_v4.x.x.sh -z
-    ```
-    A new schematics workspace job reverting the state back to its prior condition, which existed prior to the execution of the script. The script will use the `revert.json` file that got created when the script initially ran
+1.  Monitor the status of the job in the IBM Cloud [console](https://cloud.ibm.com/schematics/workspaces).
+    - When the job completes successfully, go to the next step.
+    - If the job fails, see [Reverting changes](#reverting-changes).
 
-1. After the successful upgrade to the newer release of the VSI module, it is safe to remove the temporary files generated by the script.
-    ```sh
-    rm moved.json revert.json
-    ```
+### Apply the changes in Schematics
+
+1. Update your code to consume version 4.x.x, and then update your Schematics workspace to the version of the code that contains the updated module. Click **Generate plan** and make sure none of the VSIs will be re-created. You should see changes only to names. No resources should be set to be destroyed or re-created.
+1. Click **Apply plan**.
 
-## Local Terraform State file
+    If the job is successful, follow the steps in [Clean up](#clean-up). If the job fails, see [Reverting changes](#reverting-changes).
 
-If you have both the code and the Terraform state file stored locally on your machine, you can utilize the update_v3.x.x_to_v4.x.x.sh script to avoid the recreation of Virtual Server Instances (VSIs).
+## Local Terraform
 
-1. Make sure you have all the dependencies to run the script.
-    - IBM Cloud CLI
-    - IBM Cloud CLI 'is' plugin
-    - Terraform CLI
-    - jq
+If you store both the Terraform code and state file locally, run the `update_v3.x.x_to_v4.x.x.sh` script locally.
```

Comment:
maybe it should hyperlink to the script here instead of using backticks?

Reply:
did same as for schematics. Added [View the script](schematics_update_v3.x.x_to_v4.x.x.sh).